Packs/AgariPhishingDefense/IncidentFields/incidentfield-APD_Updated_At.json

```diff
@@ -0,0 +1,30 @@
+{
+ "id": "incident_apdupdatedat",
+ "version": -1,
+ "modified": "2020-11-25T17:19:32.341520739+05:30",
+ "name": "APD Updated At",
+ "ownerOnly": false,
+ "cliName": "apdupdatedat",
```

Comment:
I would consider a different CLI name, as the current one could be a bit complex.

Reply:
We can add apdpolicyupdatedat, please suggest @guykeller